packages/utils/src/lib/utility-functions/e2e/nx.ts

```diff
@@ -8,7 +8,7 @@ export async function readDependenciesFromNxDepGraph(
   project: string,
 ) {
   const depGraphFile = join('tmp', 'dep-graph.json');
-  execSync(`npx nx dep-graph --file ${depGraphFile}`, {
+  execSync(`npx nx graph --file ${depGraphFile}`, {
```

Comment:
`graph` is probably more known than `dep-graph` by Nx newcomers (dep-graph is not documented anymore)